specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -171,6 +183,16 @@ function Merging({
     true
   );
 
+  // const [mergeId, setMergeId] = React.useState<string | undefined>(undefined);
+
+  const [aborted, setAborted] = React.useState<boolean | 'failed' | 'pending'>(
```

Comment:
try not mixing boolean and string types in the same state
why not use string types for all?
and also rename `aborted` to `state` to make it more appropriately named

Reply:
edit: looks like this state is not used at all

I am not in your situation, so feel free to disregard it, but I would advise removing unused code as soon as possible instead of commenting it out.
if you think there is a chance you would need it, then create git commits, OR separate branches OR use git stash - all of those are excellent choices.
Reasons:
- Less code in the file that way while you are in the middle of debugging/futher development - thus either to comprehent
- No risk of forgetting to remove dead code before merging to production
- Easier for people who are reviewing as they don't have to spend time reviewing dead code

Reply:
Yes I wrote it already waiting for Alec to provide me with the abort API

Reply:
"try not mixing boolean and string types in the same state
why not use string types for all?"
I took the example from the Status component we use for workbench.

Reply:
> I took the example from the Status component we use for workbench.

good reason. Workbench Mapper, Import and Status where the first React components in Specify. Despite code being refactored several times, some things are still not up to the code quality standard (i.e the use of `getState/setState` instead of `useCachedState` and the use of reducer/dispatch, and the use of custom select element, instead of a library)